doc/rados/operations/placement-groups.rst

```diff
@@ -143,16 +143,20 @@ example, a pool that maps to OSDs of class `ssd` and a pool that maps
 to OSDs of class `hdd` will each have optimal PG counts that depend on
 the number of those respective device types.
 
+In the case where a pool belongs to two or more subtrees, e.g.,
+a pool belongs to both `ssd` and `hdd` class, the autoscaler will
+issue a warning to the user in the manager log stating the name of the pool
+and the set of roots that overlap each other. The autoscaler will not
+scale any pools with overlapping roots because this can cause problems
+with the scaling process. We recommend making the pool belong to only
+one root to get rid of the warning and ensure a successful scaling process.
+
```

Comment:
This might be read as conflating device classes and CRUSH roots.

Reply:
Now that I've opened my big mouth, I think that may be more confusing.  What about the below?

In the case where a pool uses OSDs under two or more CRUSH  roots, e.g., shadow
trees with both `ssd` and `hdd` devices, the autoscaler will
issue a warning to the user in the manager log stating the name of the pool
and the set of roots that overlap each other. The autoscaler will not
scale any pools with overlapping roots because this can cause problems
with the scaling process. We recommend making the pool belong to only
one root (one OSD class) to get rid of the warning and ensure a successful
scaling process.